chart/compass/charts/gateway/templates/deployment.yaml

```diff
@@ -101,6 +108,45 @@ spec:
                   name: {{ .Values.global.auditlog.configMapName }}
                   key: auditlog-oauth-tenant-var
                   optional: true
+            {{ else if eq .Values.gateway.auditlog.authMode "oauth-mtls" }}
+            - name: APP_AUDITLOG_AUTH_MODE
+              value: "oauth-mtls"
+            - name: APP_AUDITLOG_CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.global.auditlog.secret.name }}
+                  key: {{ .Values.global.auditlog.secret.clientIdKey }}
+            - name: APP_AUDITLOG_SKIP_SSL_VALIDATION
+              value: "{{ .Values.global.auditlog.skipSSLValidation }}"
+            - name: APP_AUDITLOG_X509_CERT
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.global.auditlog.secret.name }}
+                  key: {{ .Values.global.auditlog.secret.clientCertKey }}
+            - name: APP_AUDITLOG_X509_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.global.auditlog.secret.name }}
+                  key: {{ .Values.global.auditlog.secret.clientKeyKey }}
+            - name: APP_AUDITLOG_OAUTH_URL
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.global.auditlog.secret.name }}
+                  key: {{ .Values.global.auditlog.secret.urlKey }}
+            - name: APP_AUDITLOG_TOKEN_PATH
+              value: {{ .Values.global.auditlog.tokenPath }}
+            - name: APP_AUDITLOG_OAUTH_USER
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ .Values.global.auditlog.configMapName }}
+                  key: auditlog-oauth-user-var
+                  optional: true
+            - name: APP_AUDITLOG_OAUTH_TENANT
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ .Values.global.auditlog.configMapName }}
+                  key: auditlog-oauth-tenant-var
+                  optional: true
```

Comment:
This env is populated through ConfigMap in which I didn't find such key - `auditlog-oauth-tenant-var` neither locally and in productive cluster? What is the case with this one? 

P.S. Just of curiosity - who is responsible for creating such cm in our clusters? Is it manually?

Reply:
This 'APP_AUDITLOG_OAUTH_TENANT' was not introduced from this PR. I don't know why it is marked in green.
Also, I can't find such key too.